s3watcher/lambda/lib/Transfer.js

```diff
@@ -23,25 +22,24 @@ module.exports = {
         const upload = Upload.buildUpload(config, s3Event);
 
         const success = function() {
-            Logger.logDelete(config.stage, s3Object);
-
+            logger.log('Deleting from ingest bucket.', s3Object);
             return S3Helper.deleteS3Object(s3Object);
         };
 
         const failGraceful = function(e) {
-            Logger.logImportFail(config.stage, e);
-            Logger.logCopyToFailBucket(config.stage, s3CopyObject);
+            logger.log('Import failed.', e);
```

Comment:
Oke doke, I bumped the [version to `0.1.0`](https://www.npmjs.com/package/elk-kinesis-logger), I'm happy to update and implement this after you've merged though to avoid confusion